MODULE_ACCEPTANCE_CRITERIA_TEMPLATE.MD

```diff
@@ -0,0 +1,49 @@
+* [ ] Code of Conduct statement in repository (1)
+* [ ] Installation documentation included (11)
+* [ ] Contribution guide is included in repo (13)
+* [ ] Module provides reference data (if applicable) (3, 16)
+* [ ] Personal data form is completed, accurate, and provided as PERSONAL_DATA_DISCLOSURE.md file (6)
+* [ ] Sensitive information is not checked into git repository (6)
+* [ ] Module is written in a language and framework that FOLIO development teams are familiar with _e.g. Vertx/RMB, Spring Way/folio-spring-base, and React/Stripes_ (13)
+* [ ] For backend modules: builds are Maven and JDK 11 based and Dockerfile provided (3, 5, 13)
+* [ ] Integration (API) tests written in Karate if applicable (3, 4) -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_
+* [ ] Back-end unit tests at 80% coverage (3, 4)
+* [ ] Data is segregated by tenant at the storage layer (6, 7)
+* [ ] Back-end modules don’t access data in DB schemas other than their own and public (6, 7)
+* [ ] Tenant data is segregated at the transit layer (6, 7)
+* [ ] Back-end modules respond with a tenant’s content based on x-okapi-tenant header (7)
+* [ ] Standard GET /admin/health endpoint returning a 200 response (5) -_note: read more at [https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol](https://wiki.folio.org/display/DD/Back+End+Module+Health+Check+Protocol)_
+* [ ] HA compliant (5, 14, 15)
+* [ ] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn’t been accepted yet_ (3, 5, 12)
+* [ ] Module only uses existing infrastructure / platform technologies_ e.g. PostgreSQL, ElasticSearch (and Kafka, despite it being still unofficial at present)_ (3, 5, 12)
+* [ ] Integration with any third party system (outside of the FOLIO environment) tolerates the absence of configuration / presence of the system gracefully. (3, 5, 12)
+* [ ] Front-end modules: builds are Node 16/Yarn 1 (3, 5, 13)
+* [ ] Front-end unit tests written in Jest/RTL at 80% coverage (3, 4)
+* [ ] Front-end End-to-end tests written in Cypress, if applicable  (3, 4) -_note: these tests aren’t defined as part of the module_
+* [ ] Front-end modules have i18n support via react-intl and an en.json file with English texts (8)
+* [ ] Front-end modules have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension (9)
+* [ ] Front-end modules use the current version of Stripes (10, 16)
+* [ ] Front-end modules follow relevant existing UI layouts, patterns and norms (10) -_note: read more about current practices at [https://ux.folio.org/docs/all-guidelines/](https://ux.folio.org/docs/all-guidelines/)_
+* [ ] Front end modules must work in the latest version of Chrome (the supported runtime environment) (10)
+* [ ] sonarqube hasn't identified any security issues (6)
```

Comment:
I realize that we're not debating the criteria itself in this PR, but this should probably be either "SonarQube/SonarCloud", or simply "Sonar".  I also realize that it's a bit ambiguous since we don't specify which set of sonar rules to use.